packages/dds/sequence/src/test/testFarm.ts

```diff
@@ -1126,7 +1126,7 @@ function checkInsertMergeTree(
 	const clockStart = clock();
 	mergeTree.insertSegments(
 		pos,
-		[new MergeTree.TextSegment(textSegment.text)],
+		[new MergeTree.TextSegment(textSegment.text) as MergeTree.IMergeSegment],
```

Comment:
i wish i didn't need this casting. its necessary as `BaseSegment.parent: IMergeNodeCommon` isn't implicitly convertible to `IMergeSegment.parent: IMergeBlock`